package.json

```diff
@@ -80,7 +80,7 @@
     "@komenci/contracts": "1.1.0",
     "@komenci/kit": "1.0.0-beta1",
     "@pedrouid/environment": "^1.0.1",
-    "@react-native-async-storage/async-storage": "^1.15.14",
+    "@react-native-async-storage/async-storage": "^1.17.10",
```

Comment:
is this related? if not plz split it out into a separate PR

Reply:
The package updates are part of the Statsig react native [install process](https://docs.statsig.com/client/reactNativeSDK).